zerver/openapi/zulip.yaml

```diff
@@ -5462,7 +5462,41 @@ paths:
           example: 3165826990
       responses:
         "200":
-          $ref: "#/components/responses/SimpleSuccess"
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                      detached_files:
+                        type: array
+                        description: |
+                          Returns all current attachments that were discarded after editing the message and aren't referenced in another message.
```

Comment:
This text feels wrong -- "Returns details on all uploaded files whose only references were removed when editing this message. Users should be presented with this list of messages and asked if they want to delete the uploaded files, which would otherwise remain visible only in message edit history."